storage_tiering.cpp

```diff
@@ -492,7 +494,7 @@ namespace irods {
         std::string coll_name = p.parent_path().string();
         std::string data_name = p.filename().string();
         std::string qstr{boost::str(boost::format(
-            "SELECT RESC_ID WHERE DATA_NAME = '%s' AND COLL_NAME = '%s' AND RESC_NAME IN (%s)")
+            "SELECT RESC_ID WHERE DATA_NAME = '%s' AND COLL_NAME = '%s' AND DATA_RESC_ID IN (%s)")
```

Comment:
from name to id works here?   the _partial_list is an id?

Reply:
yes, because the partial list that is passed in is now in IDs also.